.github/workflows/split_tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
                     restore-keys: ${{ matrix.php }}-comp-
 
             -   run: composer global require hirak/prestissimo
-            -   run: composer install --prefer-dist --no-progress
-            -   run: vendor/bin/monorepo-builder localize-composer-paths
-            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress
+            -   run: composer install --prefer-dist --no-progress --ansi
+            -   run: vendor/bin/monorepo-builder localize-composer-paths --ansi
+            -   run: cd packages/${{ matrix.package }}; composer update --prefer-dist --no-progress --ansi
```

Comment:
Tip for colors in CI :) 
https://tomasvotruba.com/blog/2020/09/28/how-to-add-colors-to-continuious-integration-output/